tools/unicode_properties_parse/format_width_estimate_intervals.cpp

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// The following code generates data for `_Width_estimate_intervals` in <format>.
+
+#include <charconv>
+#include <format>
+#include <fstream>
+#include <iostream>
+#include <regex>
+#include <vector>
+
+void _verify(bool test, int line, const char* msg) {
+    if (!test) {
+        std::cerr << "Error at line " << line << ": " << msg << std::endl;
+        exit(EXIT_FAILURE);
+    }
+}
+#define verify(expr, msg) _verify((expr), __LINE__, (msg))

```

Comment:
`_verify` is reserved in the global namespace.
```suggestion
void verify_impl(bool test, int line, const char* msg) {
    if (!test) {
        std::cerr << "Error at line " << line << ": " << msg << std::endl;
        exit(EXIT_FAILURE);
    }
}
#define verify(expr, msg) verify_impl((expr), __LINE__, (msg))
```

Reply:
is this reservation MSVC specific?